src/BinDeps.jl

```diff
@@ -574,18 +574,36 @@ function glibc_version()
     ismatch(Base.VERSION_REGEX, v) ? VersionNumber(v) : nothing
 end
 
+const has_sudo = Ref{Bool}(false)
+const build_source = Ref{String}("default")
+
+function __init__()
+    # install from package managers with sudo
+    has_sudo[] = if haskey(ENV, "JULIA_BINDEPS_USE_SUDO")
+        ENV["JULIA_BINDEPS_USE_SUDO"] == "true"
+    else
+        try
+            success(`sudo -V`)
```

Comment:
can you build a new system image with this line as is?  i can not.  like so:

```
$ cat ~/userimg.jl 
using BinDeps

julia> include(joinpath(JULIA_HOME, Base.DATAROOTDIR, "julia", "build_sysimg.jl"))
julia> build_sysimg("/Users/arthurb/sys.ji", "native", "/Users/arthurb/userimg.jl")

$ julia -J /Users/arthurb/sys.ji.dylib

signal (11): Segmentation fault: 11
while loading no file, in expression starting on line 0
Segmentation fault: 11
```

works though if i simply replace this line with `true`